### DISCUSSION TITLE
Different Language Versions

### DISCUSSION THREAD

**Discussion by AdamSpeight2008** [dotnet/roslyn#50205] [Q&A] [answered]
Is It possible for `VisualBasicCompilation.LanguageVersion` and `VisualBasicCompilation.Options.ParseOptions.LanguageVersion` to be different? If so which is correct?

**Comment by CyrusNajmabadi** [comment]
It's not really defined what would mean.

**Comment by AdamSpeight2008** [comment, marked as answer]
Submitted https://github.com/dotnet/roslyn/pull/50206
